onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -308,6 +309,26 @@ TEST(OrtModelOnlyTests, SerializeToOrtFormatMLOps) {
 #endif  // #if !defined(DISABLE_ML_OPS)
 #endif  // #if !defined(ORT_MINIMAL_BUILD)
 
+TEST(OrtModelOnlyTests, SparseInitializerHandling) {
+  const std::basic_string<ORTCHAR_T> ort_file = ORT_TSTR("sparse_initializer_handling.onnx.ort");
+  SaveAndCompareModels("testdata/sparse_initializer_handling.onnx", ort_file);
```

Comment:
if testdata/sparse_initializer_handling.onnx was generated with some script, is it possible to check that in too?

Reply:
It was not generated with a script.

---
In reply to: [508946570](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r508946570) [](ancestors = 508946570)